Automattic/jetpack
# Contact Form: add deprecation warnings to Contact Form module codebase

Fixes https://github.com/Automattic/jetpack/issues/35654

## Proposed changes:

Add deprecation warnings to the global functions, classes, public methods, and properties in the Contact Form module codebase. The code has been brought over to the Forms package.

This PR also updates some Jetpack plugin tests to reference the Forms package instead of the Contact Form module. Indeed, the deprecation warnings added to the module made many tests fail.

### Other information:


## Jetpack product discussion


Project thread: pf5801-Aj-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






There's not much to test here.

- Spin up a test site with this branch
- Make sure you have access to the server logs
- Use the Contact Form module instead of the Forms package by ensuring the condition at this line is falsy: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/modules/contact-form.php#L33
- Verify you see the deprecation warnings in the logs
